test/uk/gov/hmrc/helptosave/repo/MongoEmailStoreSpec.scala

```diff
@@ -19,14 +19,10 @@ package uk.gov.hmrc.helptosave.repo
 import org.scalatest.concurrent.Eventually
 import play.api.libs.json.{JsObject, JsString}
 import play.modules.reactivemongo.ReactiveMongoComponent
-import reactivemongo.api.indexes.Index
 import reactivemongo.play.json.collection.JSONCollection
-import uk.gov.hmrc.helptosave.repo.MongoEmailStore.EmailData
 import uk.gov.hmrc.helptosave.util.{Crypto, NINO}
 import uk.gov.hmrc.helptosave.utils.TestSupport
 
-import scala.concurrent.duration._
-import scala.concurrent.Await
 import scala.util.{Failure, Success, Try}
 
 class MongoEmailStoreSpec extends TestSupport with Eventually with MongoSupport {
```

Comment:
do we still need `Eventually` because it was added for embedmongo ?

Reply:
there's a reference to eventually in this spec

Reply:
i see that, but thats also not needed i guess, but anyway its not a big deal , thats fine if we have it

Reply:
what we found before was without the eventually, if we did a get immediately after inserting it was failing to find the data we had just inserted - with eventually it retries until it's successful. I'll give it a go to see if it works without the eventually now

Reply:
just gave it a go and I saw the problem I described above

Reply:
thats probbabably because we are not waiting for the previous `remove` future to complete